sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/FollowUpVisitService.java

```diff
@@ -110,4 +110,11 @@ public Visit buildSpecifiedFollowUpVisitForAvailableAndCooperative() {
         .sourceOfBodyTemperature(getRandomTemperature())
         .build();
   }
+
+  public Visit buildSpecifiedSymptoms() {
```

Comment:
let's rename this to point only to temperature data, and pass the degrees as parameter so we can vary it in code however we need for further scenarios